packages/http-server/src/__tests__/server.oas.spec.ts

```diff
@@ -206,6 +206,30 @@ describe.each([['petstore.oas2.json'], ['petstore.oas3.json']])('server %s', fil
     expect(response.statusCode).toBe(500);
     checkErrorPayloadShape(response.payload);
   });
+
+  test('should mock the response headers', async () => {
+    const response = await server.fastify.inject({
+      method: 'GET',
+      url: '/user/login?username=foo&password=foo',
+    });
+
+    // OAS2 does not support examples for Headers, to they MUST be always generated automagically,
```

Comment:
Does graphite, and therefore prism, support `x-example` for OAS2? If so it would be ok to have default behavior tested like this and then a targeted unit test showing the support.

Reply:
That can be a follow up feature btw no need to scope creep this one.

Reply:
At the moment no, x-example is not supported in graphite.